apps/builder/app/builder/features/settings-panel/shared.tsx

```diff
@@ -149,12 +150,15 @@ export const VerticalLayout = ({
 export const HorizontalLayout = ({
   label,
   id,
+  labelSize,
   onDelete,
   children,
 }: LayoutProps) => (
   <Grid
     css={{
-      gridTemplateColumns: `${theme.spacing[19]} 1fr`,
+      gridTemplateColumns: `${
+        labelSize === "default" ? theme.spacing[19] : theme.spacing[24]
```

Comment:
so "default" is not the default, because if you omit the prop, you will get "large"